From e05049afbb37dd196498e22076d155a28ae654d2 Mon Sep 17 00:00:00 2001
From: dvshkurk <Dmitry.V.Shkurko@intel.com>
Date: Tue, 18 Dec 2012 20:02:23 +0700
Subject: [PORT FROM MAIN][BT] Likely incorrect assignment if-statement

BZ: 79142

Change-Id: I998080a05bc396aa1e28137a3555e29d822b0189
Orig-Change-Id: Ice25a992bd281778eba06eb286f75522364d2e1c
Feature: BT
Category: Integration
Purpose: PV-Enabler
Origin: Internal
Upstream-candidate: yes
Signed-off-by: dvshkurk <Dmitry.V.Shkurko@intel.com>
Reviewed-on: http://android.intel.com:8080/85713
Reviewed-by: Shkurko, Dmitry V <dmitry.v.shkurko@intel.com>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 btif/src/btif_hl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/btif/src/btif_hl.c b/btif/src/btif_hl.c
index e76c0c3..8831c9b 100644
--- a/btif/src/btif_hl.c
+++ b/btif/src/btif_hl.c
@@ -3492,7 +3492,7 @@ static void btif_hl_upstreams_evt(UINT16 event, char* p_param){
                     if (btif_hl_find_mcl_idx(app_idx, p_data->sdp_query_cfm.bd_addr, &mcl_idx))
                     {
                         p_mcb = BTIF_HL_GET_MCL_CB_PTR(app_idx, mcl_idx);
-                        if ( (p_mcb->cch_oper =  BTIF_HL_CCH_OP_MDEP_FILTERING) ||
+                        if ( (p_mcb->cch_oper == BTIF_HL_CCH_OP_MDEP_FILTERING) ||
                              (p_mcb->cch_oper == BTIF_HL_CCH_OP_DCH_OPEN) )
                         {
                             pending_op = p_mcb->pcb.op;
-- 
1.7.4.1

